### DISCUSSION TITLE
也许可以试试GLM-3-Turbo，成本是GPT3.5T的一半左右，作者能多赚不少钱😂

### DISCUSSION THREAD

**Discussion by Diplproveably** [immersive-translate/immersive-translate#1407] [Ideas]
### 新功能 | New Feature

chatGPT的API是输入$0.0005 / 1K tokens输出$0.0015 / 1K tokens。然后chatGPT的Tokenizer让平均输入一个英文Token会翻译出来两个中文Token左右。
这样如果翻译1M token的开销就3.5美元，大概25RMB。

然后如果是GLM-3-Turbo的话，输入输入一个价格都是0.005元 / 千tokens，GLM的Tokenizer对中文的优化更好，平均一个英文Token翻译到中文基本上也是一个Token，这样翻译1M文本就是10RMB。

而且智谱的盘子比openai小点，可能议价也比openai容易点😂

如果是云端部署GLM3的话可能还能更便宜点……

**Comment by theowenyoung** [comment]
智谱的API，我们测试过，翻译一篇外网的文章，36%的内容提示内容敏感，不允许翻译...
